config/routes.rb

```diff
@@ -12,9 +16,10 @@
       end
       resources :manifests, only: [:create]
       resources :method_codes, only: [:index]
-      patch 'manifests', to: 'manifests#update'
-      get 'manifest', to: 'manifests#show'
+      patch 'manifests/:id', to: 'manifests#update'
+      put 'manifests/:id', to: 'manifests#update'
```

Comment:
oh, we don't. I got ambitious. Since we don't really do a replace-in-place like a proper PUT, I will remove it.